src/osx/core/uilocale.mm

```diff
@@ -185,6 +226,104 @@ int CompareStrings(const wxString& lhs, const wxString& rhs,
     return wxCFStringRef::AsString(str);
 }
 
+wxString
+wxUILocaleImplCF::GetMonthName(wxDateTime::Month month, wxDateTime::NameForm form) const
+{
+    const int idx = ArrayIndexFromFlag(form.GetFlags());
+    if (idx == -1)
+        return wxString();
+
+    NSDateFormatter* df = [NSDateFormatter new];
+    df.locale = m_nsloc;
+
+    NSArray* monthNames = nullptr;
+
+    if (form.GetContext() == wxDateTime::Context_Standalone)
+    {
+        switch (idx)
```

Comment:
It seems strange to me to convert flags to index when we don't use it as an index, finally. And having `case wxDateTime::Name_Full` etc would be more clear than the code we have here now.